lib/aryn-sdk/aryn_sdk/partition/partition.py

```diff
@@ -183,6 +203,7 @@ def _json_options(
     extract_images: bool = False,
     selected_pages: Optional[list[Union[list[int], int]]] = None,
     output_format: Optional[str] = None,
+    chunking_options: Optional[dict[str, Any]] = None,
```

Comment:
we handle malformed chunking options well in the service, yes?

Reply:
Yes.